browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -819,6 +819,9 @@ until they verify, or until 90 days have passed.
       <message name="IDS_ACCESSIBILITY_TOOLBAR_BTN_BOOKMARK" desc="Content description for the bookmark button.">
         Create a bookmark
       </message>
+      <message name="IDS_BRAVE_SYNC_V1_DEPRECATED_INFOBAR_MESSAGE" desc="Message text shown to users who used a previous version of Brave Sync and need to learn about a new version coming soon.">
```

Comment:
can we re-use the desktop string, or they are not included for android?

Reply:
@petemill 
they are not included in Android, https://bravesoftware.slack.com/archives/C0816QH3R/p1595256920155200